packages/marko-web-html-sitemap/templates/date-list.marko

```diff
@@ -0,0 +1,60 @@
+$ const { config, GAM } = out.global;
+$ const { nodes, displayType, month, year, alias } = input;
+
+$ const type = "sitemap";
+$ const pageNode = {
+  title: 'Sitemap',
+};
+$ switch (displayType) {
+  case "day":
+      pageNode.description = `Select from the list of days in ${month} ${year}`;
+      break
+  case "month":
+      pageNode.description = `Select from the list of months in ${year}`;
+      break
+  default:
+    pageNode.description = `Select from the list of years`;
```

Comment:
This should have translation support like the inquiry package now does. ~The date format (and language) should be pulled from the site config also (it's customized on at least one client...maybe bizbash?)~
Nevermind this part -- the es-mx date format would be the same.

Reply:
@solocommand I am going to do a second PR for adding in lang support